shared/src/main/java/com/couchbase/lite/FileLogger.java

```diff
@@ -167,7 +71,11 @@ public void setDirectory(String directory) {
      *
      * @param level The maximum level to include in the logs
      */
-    public void setLevel(LogLevel level) {
+    public void setLevel(LogLevel level) throws IllegalStateException {
```

Comment:
For consistency, we have never declared to throw IllegalStateException for null checking. I think we should remove it for consistency.

Reply:
Interesting.  I thought it was required to add this.....there is much I don't know about Java.  I'll try removing this stuff and see if it complains.

Reply:
It's not required to declare runtime exception as it's not required to catch it. I will remove the declaration and push to this PR so that we can just merge.